File: ProblemSet_34xx/Problem_3477/solution.js
/**
 * @param {number[]} fruits
 * @param {number[]} baskets
 * @return {number}
 */
var numOfUnplacedFruits = function(fruits, baskets) {
    var result = 0;

    fruits.forEach(fruit => {
        var isPlaced = false;
        for(var i = 0; i<baskets.length; i++){
            if (baskets[i] >= fruit) {
                isPlaced = true;
                baskets[i] = 0;
                break;
            }
        }
        if (!isPlaced) result += 1;
    });

    return result;
};

File: ProblemSet_34xx/Problem_3477/solution.py
# Write your solution here
from typing import List


class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        result = 0
        for fruit in fruits:
            isPlaced = False
            for j in range(len(baskets)):
                if (baskets[j] >= fruit):
                    baskets[j] = 0
                    isPlaced = True
                    break
            if (not isPlaced):
                result += 1
        return result
                